### ISSUE TITLE
Bounce hangs with VDiskIn (if too long?)

### ISSUE THREAD

**Issue by Sciss** [Sciss/SoundProcesses#74]
In GitLab by @Sciss on Apr 19, 2020, 14:01

```scala
val factor = XLine.ar(1.0/16, 1.0, 30 * 60)
val in = VDiskIn.ar("in", speed = factor, interp = 2 /* 4 */, maxSpeed = 1.0)
factor.roundTo(0.01).poll(30.0.reciprocal, "f")
Out.ar(0, in)
```
Bouncing this in rt for 30mins with a 11min stereo input file hangs at the end. Both in normalized/non-normalized. 48 kHz.

**Comment by Sciss** [comment]
In GitLab by @Sciss on Apr 19, 2020, 14:20

The issue is not the length of the file, but the case where the buffer reader switches to `b_fill` after the file is exhausted. After that moment, we're missing an `n_free` for the main synth, although a `removeVertex` is still observed.

**Comment by Sciss** [comment]
In GitLab by @Sciss on Apr 19, 2020, 15:41

mentioned in commit 3c08b13e2525be50ad20b717ce5c56461f983dd3

**Comment by Sciss** [comment]
In GitLab by @Sciss on Apr 19, 2020, 15:41

closed via commit 3c08b13e2525be50ad20b717ce5c56461f983dd3
